src/app/(views)/enrolled-courses/page.tsx

```diff
@@ -21,7 +21,9 @@ export default function ManagedCourses() {
 
       <div className="d-grid gap-4" style={{ gridTemplateColumns: 'repeat(auto-fill, minmax(200px, 1fr))', justifyContent: 'left' }}>
         {fetchingCourses ? (
+          <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '60vh' }}>
```

Comment:
If you're going to add ` <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '60vh' }}>` everytime, you might as well put it inside `<Loading />`,  no?

Reply:
true, but I have yet to explore find where that it

Reply:
kita ko na